src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+let div;
+const logo = document.querySelector('.logo');
+const promise1 = new Promise(function(resolve, reject) {
+  logo.addEventListener('click', () => {
+    resolve(`Promise was resolved!`);
+  });
+});
+
+promise1.then(resolution => {
+  div = document.createElement('div');
+  div.classList.add('message');
+  div.textContent = resolution;
+  document.body.append(div);
+}).catch(Error('error!'));
+
+const promise2 = new Promise(function(resolve, reject) {
+  setTimeout(() => reject(new Error(`Promise was rejected!`)), 3000);
+});
+
+promise2.then(() => {
+  return 'Succes';
+})
+  .catch(rejection => {
+    div = document.createElement('div');
+    div.classList.add('error-message');
+    div.textContent = rejection;
+    document.body.append(div);
```

Comment:
These functions has a lot in common. They only differs with message and text. Let's write an addMessage function that takes two args `message` and `type`